answerfile.py

```diff
@@ -101,6 +101,8 @@ def processAnswerfile(self):
             else:
                 raise AnswerfileException("Unknown mode, %s" % install_type)
 
+            results['gpgcheck'] = getBoolAttribute(self.top_node, ['gpgcheck'], default = True)
+            results['repo-gpgcheck'] = getBoolAttribute(self.top_node, ['repo-gpgcheck'], default = True)
```

Comment:
Regarding the coding style, spaces are not used to reference a named param. Here `default = True`. ;)

Reply:
Right, I should not have kept that space from the 8.2.1 code  ;)